src/summon/browzine-summon-adapter.js

```diff
@@ -505,7 +505,7 @@ browzine.summon = (function() {
     var articlePDFDownloadWording = "View Now";
     var articlePDFDownloadLinkText = "PDF";
 
-    if (browzine.version >= 2) {
+    if (+browzine.version >= 2) {
```

Comment:
TIL about the unary `+` operator: https://medium.com/@nikjohn/cast-to-number-in-javascript-using-the-unary-operator-f4ca67c792ce

I also checked to see what happens if `version` was set to something like "2a", "tbd", etc., and I see `+browzine.version >= 2` should evaluate to falsy:

![CleanShot 2021-12-13 at 08 59 53@2x](https://user-images.githubusercontent.com/1091559/145835798-cb6b320d-c39e-4379-81e9-c8313d9a2ff6.png)

Only thing I can think of to maybe add a test - which we could do after merging this PR - is writing a test to define the behavior when `version` is set to an alphanumeric like that or other invalid value.